webapi/src/main/scala/org/knora/webapi/util/ConstructResponseUtilV2.scala

```diff
@@ -722,7 +721,7 @@ object ConstructResponseUtilV2 {
                     case (propIri: SmartIri, values: Seq[ValueRdfData]) =>
 
                         // check if the property Iri already exists (there could be several instances of the same property)
-                        if (acc.get(propIri).nonEmpty) {
+                        if (acc.contains(propIri)) {
```

Comment:
I think this change could have been automated :-)

Reply:
It was automated. :)